hack/update-codegen.sh

```diff
@@ -24,6 +24,13 @@ fi
 
 source $(dirname $0)/../vendor/github.com/knative/test-infra/scripts/library.sh
 
+
+# Clone the v1beta1 types to v1alpha1 sans v1beta1.
+for x in $(ls "${REPO_ROOT_DIR}/config/v1beta1"); do
+  head -n27 "${REPO_ROOT_DIR}/config/v1beta1/$x" > "${REPO_ROOT_DIR}/config/v1alpha1/$x"
```

Comment:
It you are going to do line numbers then `sed -e '28,30d'` brings this to a single line instead of head and tail.